worksheets.go

```diff
@@ -684,13 +684,18 @@ func (ws *Worksheet) handleDependentUpdates(field *Field, oldValue, newValue Val
 }
 
 func extractChildWs(value Value) []*Worksheet {
-	if childWs, ok := value.(*Worksheet); ok {
-		return []*Worksheet{childWs}
-	}
-	if _, ok := value.(*Slice); ok {
-		panic("slices-of-slices are not supported yet")
```

Comment:
from #85 
>though when we merge, we should test slices of slices!

Reply:
figured I'd leave this as open since we have #32 🤷‍♂️

Reply:
Ah ok, makes sense. Didn't know about that one.

Reply:
Ah ok, makes sense. Didn't know about that one.